lua/grapple-utils.lua

```diff
@@ -22,6 +13,58 @@ local function get_current_index(tags, current_path)
 	return nil
 end
 
+function M.move_to_index(opts)
+	if not opts or not opts.index then
+		vim.notify("Invalid options: index is required", vim.log.levels.ERROR)
+		return
+	end
+
+	local current_path = vim.api.nvim_buf_get_name(0)
+	local current_opts = { path = current_path }
+	if not grapple.exists(current_opts) then
+		vim.notify("Current buffer is not tagged", vim.log.levels.INFO)
+		return
+	end
+
+	local tags, err = grapple.tags()
+	if not tags then
+		vim.notify("No tags available: " .. err, vim.log.levels.ERROR)
```

Comment:
This should never happen, so should be an actual lua error or `assert` rather than a `vim.notify`.